docs/user/sbt.rst

```diff
@@ -39,8 +39,6 @@ Sbt settings
 ======================== =============== =======================================
 Name                     Type            Description
 ======================== =============== =======================================
-``nativeClang``          ``File``        Path to ``clang`` command
-``nativeClangPP``        ``File``        Path to ``clang++`` command
 ``nativeCompileOptions`` ``Seq[String]`` Extra options passed to clang verbatim
```

Comment:
The problem with the task is that you cannot set it.
`set nativeMode := "release"`